_maps/Configs/nanotrasen_tanker.json

```diff
@@ -0,0 +1,33 @@
+{
+	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
+	"prefix": "NTSV",
```

Comment:
This will need to be "NEU", as, tragically, we only have three accepted prefixes here.